sphinx-favicon/__init__.py

```diff
@@ -71,7 +71,9 @@ def _static_to_href(pathto: Callable, favicon: Dict[str, str]) -> Dict[str, str]
     """
     if FILE_FIELD in favicon:
         attrs = favicon.copy()
-        attrs["href"] = pathto(f"_static/{attrs.pop(FILE_FIELD)}", resource=True)
+        attrs["href"] = pathto(
+            f"{OUTPUT_STATIC_DIR}/{attrs.pop(FILE_FIELD)}", resource=True
```

Comment:
I updated this line to use the static variable `OUTPUT_STATIC_DIR` instead of `"_static"`. @abravalheri Is there a reason why you defined `OUTPUT_STATIC_DIR` in line 11 but then didn't use it here?

Reply:
Sorry, I probably forgot 😅

Reply:
No problem! 😀